services/network/setup.py

```diff
@@ -13,4 +13,8 @@
       version='0.1.0',
       description='Wifi manager for wpa_supplicant',
       license='MIT',
-      install_requires=['bottle == 0.12.8'])
+      install_requires=[
+        'binascii',
+        'bottle == 0.12.8',
+        'hashlib'
```

Comment:
Remember to add the version of the new packages this will avoid any problems related to updates that may break the dependency api.

Reply:
thanks. it turns out these are distributed with python, so I'll take them out of here
https://docs.python.org/3/library/